dependencies.yml

```diff
@@ -89,195 +30,29 @@ collectors:
-  - type: repo-issue
-    versions: "Y.0.0"
-
-- type: js-npm
-  path: lib/cli
-  actors:
-  # pull requests for updates to our major version
-  - type: js-npm
-    versions: "L.Y.Y"
-  # create issues for new major versions
-  - type: repo-issue
-    versions: "Y.0.0"
-
-- type: js-npm
-  path: lib/channels
-  actors:
-  # pull requests for updates to our major version
-  - type: js-npm
-    versions: "L.Y.Y"
-  # create issues for new major versions
-  - type: repo-issue
-    versions: "Y.0.0"
-
-- type: js-npm
-  path: lib/addons
-  actors:
-  # pull requests for updates to our major version
-  - type: js-npm
-    versions: "L.Y.Y"
-  # create issues for new major versions
-  - type: repo-issue
-    versions: "Y.0.0"
-
-- type: js-npm
-  path: addons/info
-  actors:
-  # pull requests for updates to our major version
-  - type: js-npm
-    versions: "L.Y.Y"
-  # create issues for new major versions
-  - type: repo-issue
-    versions: "Y.0.0"
-
-- type: js-npm
-  path: addons/centered
-  actors:
-  # pull requests for updates to our major version
-  - type: js-npm
-    versions: "L.Y.Y"
-  # create issues for new major versions
-  - type: repo-issue
-    versions: "Y.0.0"
-
-- type: js-npm
-  path: addons/knobs
-  actors:
-  # pull requests for updates to our major version
-  - type: js-npm
-    versions: "L.Y.Y"
-  # create issues for new major versions
-  - type: repo-issue
-    versions: "Y.0.0"
-
-- type: js-npm
-  path: addons/graphql
-  actors:
-  # pull requests for updates to our major version
-  - type: js-npm
-    versions: "L.Y.Y"
-  # create issues for new major versions
-  - type: repo-issue
-    versions: "Y.0.0"
-
-- type: js-npm
-  path: addons/notes
-  actors:
-  # pull requests for updates to our major version
-  - type: js-npm
-    versions: "L.Y.Y"
-  # create issues for new major versions
-  - type: repo-issue
-    versions: "Y.0.0"
-
-- type: js-npm
-  path: addons/comments
-  actors:
-  # pull requests for updates to our major version
-  - type: js-npm
-    versions: "L.Y.Y"
-  # create issues for new major versions
-  - type: repo-issue
-    versions: "Y.0.0"
-
-- type: js-npm
-  path: addons/actions
-  actors:
-  # pull requests for updates to our major version
-  - type: js-npm
-    versions: "L.Y.Y"
-  # create issues for new major versions
-  - type: repo-issue
-    versions: "Y.0.0"
-
-- type: js-npm
-  path: addons/storyshots
-  actors:
-  # pull requests for updates to our major version
-  - type: js-npm
-    versions: "L.Y.Y"
-  # create issues for new major versions
-  - type: repo-issue
-    versions: "Y.0.0"
-
-- type: js-npm
-  path: addons/options
-  actors:
-  # pull requests for updates to our major version
-  - type: js-npm
-    versions: "L.Y.Y"
-  # create issues for new major versions
-  - type: repo-issue
-    versions: "Y.0.0"
-
-- type: js-npm
-  path: addons/links
+- type: js-lerna
+  path: /
+  settings:
+    bootstrap_command: yarn && yarn bootstrap -- --core
```

Comment:
I think we want to hold this until #1810

Reply:
Ok, I'll add both to 3.3 milestone (see https://github.com/storybooks/storybook/pull/1803#issuecomment-327573050)

Reply:
In fact, the command has to stay the same: the bootstrap script won't work without root depandencies